build-system/app.js

```diff
@@ -552,6 +552,10 @@ app.post('/get-consent-v1/', (req, res) => {
   assertCors(req, res, ['POST']);
   const body = {
     'promptIfUnknown': true,
+    'sharedDataPromise': {
```

Comment:
`sharedDataPromise`? I thought we name it `sharedData`?

Reply:
good catch. blame my IDE :-)